SecureSubmit.php

```diff
@@ -974,10 +974,13 @@ function getPanel(panelName) {
 
                         additionalNext.on("click", function(event) {
                             var continueProcessing = true;
-                            frameBody.find('.required').each(function(i, obj) {
+                            additionalPanel.find('.required').each(function(i, obj) {
                                 if (jQuery(this).val() == '' || jQuery(this).val() == 'Select an option below') {
+                                    var thisEle = jQuery(this);
+                                    var elementText = thisEle.attr("placeholder");
```

Comment:
theoretically i would identify the field by its placeholder. but i wasn't sure that applied in additional info and forgot to remove this reference